src/gui/render/gl_enums.hpp

```diff
@@ -114,7 +424,7 @@ get_size(const GPUDataType& data_type) {
         case GPUDataType::DOUBLE:
             return 8;
         default:
-            abort();
+            abort(); // ya this is not grate
```

Comment:
Why

Reply:
I think it should include all the types. Only 8 of the 110 types are implemented.

Reply:
then edit your comment :)